inlong-sdk/sort-sdk/src/main/java/org/apache/inlong/sdk/sort/manager/InlongMultiTopicManager.java

```diff
@@ -0,0 +1,289 @@
+        for (int i = 0; i < consumerSize; i++) {
+            try {
+                PulsarClient pulsarClient = PulsarClient.builder()
+                        .serviceUrl(topic.getInLongCluster().getBootstraps())
+                        .authentication(AuthenticationFactory.token(topic.getInLongCluster().getToken()))
+                        .build();
+                TopicFetcher fetcher = TopicFetcherBuilder.newPulsarBuilder()
+                        .pulsarClient(pulsarClient)
+                        .topic(topics)
+                        .context(context)
+                        .subscribe();
+                fetchers.add(fetcher);
+            } catch (PulsarClientException e) {
+                LOGGER.error("failed to create pulsar client for {}\n", topic.getInLongCluster().getBootstraps(), e);
+            }
+        }
+        fetchers.forEach(topicFetcher -> allFetchers.put(topicFetcher.getFetchKey(), topicFetcher));
+    }
+
+    private void updateTubeFetcher(String  clusterId, List<InLongTopic> topics) {
+        List<TopicFetcher> fetchers = tubeFetchers.computeIfAbsent(clusterId, k -> new ArrayList<>());
+        if (CollectionUtils.isNotEmpty(fetchers)) {
+            fetchers.forEach(fetcher -> fetcher.updateTopics(topics));
+            return;
+        }
+        InLongTopic topic = topics.stream().findFirst().get();
+        LOGGER.info("create new tube multi topic consumer for bootstrap {}, size is {}",
+                topic.getInLongCluster().getBootstraps(), consumerSize);
+        for (int i = 0; i < consumerSize; i++) {
+            try {
+                TubeClientConfig tubeConfig = new TubeClientConfig(topic.getInLongCluster().getBootstraps());
+                MessageSessionFactory messageSessionFactory = new TubeSingleSessionFactory(tubeConfig);
+                TubeConsumerCreator tubeConsumerCreator = new TubeConsumerCreator(messageSessionFactory,
+                        tubeConfig);
+                topics.forEach(tubeTopic -> {
+                    TopicFetcher fetcher = TopicFetcherBuilder.newTubeBuilder()
+                            .tubeConsumerCreater(tubeConsumerCreator)
+                            .topic(tubeTopic)
+                            .context(context)
+                            .subscribe();
+                    fetchers.add(fetcher);
+                });
+            } catch (TubeClientException e) {
+                LOGGER.error("failed to create tube client for {}\n", topic.getInLongCluster().getBootstraps(), e);
+            }
+        }
+        fetchers.forEach(topicFetcher -> allFetchers.put(topicFetcher.getFetchKey(), topicFetcher));
+    }
+
+    private class UpdateMetaDataThread extends PeriodicTask {
+
+        public UpdateMetaDataThread(long runInterval, TimeUnit timeUnit) {
+            super(runInterval, timeUnit, context.getConfig());
+        }
+
+        @Override
+        protected void doWork() {
+            logger.debug("InLongTopicManagerImpl doWork");
```

Comment:
logger need to upper case.

Reply:
The logger is inherited from base class ***PeriodicTask***, and it is not a static variables